src/main/java/frc/robot/Constants.java

```diff
@@ -129,12 +129,7 @@
 
     //#region Climber Details
 
-    public static final double CLIMBER_POT_SCALE_VALUE = 10.0;
-
-    public static final double CLIMBER_POT_MAX_DELTA = 1;
-    public static final double CLIMBER_POT_MIN = 0;
-
-    public static final double CLIMBER_MAX_PERCENT_OUTPUT = 0.5;
```

Comment:
As far as I'm aware, most of these Constants are still relevant, and should probably not be deleted